Airports/EGCC/Sector.txt

```diff
@@ -79,7 +79,7 @@ DISPLAY:CCSTH05:Scottish PC East:CCSTH05
 DISPLAY:CCFIN05:CCFIN05:CCSTH05
 DISPLAY:CCSTH05:CCFIN05:CCSTH05
 COORD:N053.21.25.000:W002.15.44.000
-COORD:N053.16.47.000:W001.46.46.000
+COORD:N053.14.20.412:W002.01.44.903
```

Comment:
Let's make these coordinates match up exactly while we're at it! This point should be overhead DAYNE, so:

```suggestion
COORD:N053.14.19.000:W002.01.45.000
```

I'd also suggest changing all instances of `N053.14.18.990 W002.01.45.000` to `N053.14.19.000 W002.01.45.000` (i.e. the coordinates of DAYNE) in RMA.txt